architecture/infrastructure_components/web_console.adoc

```diff
@@ -50,6 +50,22 @@ whitelist that host name by specifying the `--cors-allowed-origins` option
 on `openshift start` or from the related
 xref:../../install_config/master_node_configuration.adoc#master-configuration-files[master
 configuration file parameter `corsAllowedOrigins`].
+
+The `corsAllowedOrigins` parameter is fully under the control of the
+configuration field. No pinning or escaping is done to the value. Here is an
+example of how you can pin a host name and escape dots:
```

Comment:
Because "fully" is an abverb, can I sugget: "The `corsAllowedOrigins` parameter is controlled by the configuration field." Though I'm not too sure which field it's talking about. the example below? Also, my suggestion is passive, but I think it's acceptable because we're talking about the parameter, and that should remain being the object.

Also, instead of "Here is..." can I suggest simply: "To pin a host name and escape dots" (Though I'm not too sure if I'm suggesting the right words). Up to you!